i18n/en/index.ts

```diff
@@ -13,4 +13,5 @@ export const English = {
   signup: 'Sign Up',
   welcome: 'Welcome',
   info: 'Info',
+  searchPlaceholder: 'search for games or players…',
```

Comment:
Player search isn't planned yet.

Reply:
Yeah I know, but we also have it in the [new design](https://www.figma.com/design/g17hPsrsOqGPrD94iDxiyF/Leaderboard.gg?node-id=303-1351&t=hNEMXLYpypz8M01L-0). Which also as we've discussed in the server, this design does clash with the steering decision. I think there's a bigger discussion in sorting out what our design really should be in the first place.